src/components/card/Card.jsx

```diff
@@ -1,82 +1,64 @@
+    this.getAssignees = this.getAssignees.bind(this)
   }
 
-  handleChangeAssignee(value) {
-    const memberId = value[value.length - 1].slice(this.props.id.length, value.toString().length)
-    if (this.props.card.assignees.map(element => element.id).includes(memberId) === true) {
-      this.props.removeAssigneeInCard(this.props.card.id, memberId)
-    } else {
-      this.props.addAssignee(this.props.card.id, memberId)
-    }
+  getAssignees() {
+    return (
+      <AssigneesContainer
+        cardId={this.props.id}
+        target="cardDetails"
+        displayAssignees
+        displaySelectAssignees
+        displaySelectResponsible
+      />
+    )
   }
 
-  handleChangeLabels(value) {
-    const labelId = value[value.length - 1].slice(this.props.id.length, value.toString().length)
-    if (this.props.card.labels.map(element => element.id).includes(labelId) === true) {
-      this.props.removeLabelInCard(this.props.card.id, labelId)
-    } else {
-      this.props.addLabel(this.props.card.id, labelId)
-    }
+  getLabels() {
+    return (
+      <LabelsContainer cardId={this.props.id} displayLabels displaySelect />
+    )
   }
 
   render() {
     return (
       <div>
-        <Layout>
-          <Layout>
-            <Content className="content">
-              <EditArea
-                text={this.props.card.desc}
-                save={(newDesc) => { this.props.updateDesc(this.props.id, newDesc) }}
-                hint="Add a description"
-              />
-              <p>Desc : {this.props.card.desc}</p>
-              <p>DueDate : {this.props.card.dueComplete}</p>
-              <p>Pos : {this.props.card.pos}</p>
-              <p>List ID : {this.props.card.listId}</p>
-              <p>ID : {this.props.id}</p>
-              <p>More...</p>
-            </Content>
-            <Sider className="sider">
-              <Select
-                mode="tags"
-                style={{ width: '60%' }}
-                placeholder="Select users..."
-                onChange={this.handleChangeAssignee}
-                defaultValue={this.props.card.assignees.map(assignee => assignee.fullName)}
-                tokenSeparators={[',']}
-              >
-                {this.props.users.map(user => <Option key={this.props.id + user.id}>{user.fullName}</Option>)}
-              </Select>
-              <Select
-                mode="tags"
-                style={{ width: '60%' }}
-                placeholder="Select labels..."
-                onChange={this.handleChangeLabels}
-                defaultValue={this.props.card.labels.map(label => label.name)}
-                tokenSeparators={[',']}
-              >
-                {this.props.labels.map(label => <Option key={this.props.id + label.id}>{label.name}</Option>)}
-              </Select>
-              <Button type="primary" className="siderButton">Button drive</Button>
-            </Sider>
-          </Layout>
+        <Layout><Layout>
+          <Content className="content">
```

Comment:
Be careful with a such basic class name, it could be overloaded by another piece of code using the same class name and causing some side effects. Prefer something like `cardContent`.